ocaml/tests/test_storage_migrate_state.ml

```diff
@@ -32,7 +32,7 @@ let sample_send_state =
           ; vdi= Storage_interface.Vdi.of_string "mirror_vdi"
           ; url= "remote_url"
           }
-    ; local_dp= "local_dp"
+    ; local_dp= "remote_dp"
```

Comment:
By the looks of it there are a few pieces of clean up i should do in this instance